scripts/run-tests.sh

```diff
@@ -28,29 +28,51 @@ function usage() {
 
   color yellow "Options:"
 
-  color green "  --update"
+  color green "  -u, --update"
   echo -e "\tUpdate goldens during integration tests."
 
+  color green "  -g, --update-goldens"
```

Comment:
what's the difference with `color green "  -u, --update"` ? 

Update echo below or remove duplicate.

Reply:
fixed